examples/objectDetection-video/sketch.js

```diff
@@ -0,0 +1,61 @@
+/*
+* 👋 Hello! This is an ml5.js example made and shared with ❤️.
+* Learn more about the ml5.js project: https://ml5js.org
+* ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+*
+* This example demonstrates detecting objects in a video file through ml5.objectDetector.
+*/
+
+let video;
+let detector;
+let detections = [];
+
+function preload(){
+  detector = ml5.objectDetector("cocossd");
+}
+
+// Callback function is called each time the object detector finishes processing a frame.
+function gotDetections(results) {
+  // Update detections array with the new results
+  detections = results;  
+}
+
+function setup() {
+  // Create canvas with initial size - will be resized later
+  createCanvas(640, 480);
+  
+  // Load and loop the video for object detection
+  video = createVideo('test.mov');
+  video.size(width, height);
```

Comment:
Just noting our discussion to use a video we have permission for. Also, I would suggest sizing the video to 640x480 so it doesn't appear squashed or skewed. Or just make the canvas size identical to the video. You can add a code comment that explains that if the video does not match the canvas size, you may need to adjust the code to scale.